app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
```

Comment:
Use annotation, it is a good practice (https://github.com/bakutake/py-lockdown-cafe-test_task/blob/master/checklist.md):

```python3
def go_to_cafe(friends: list, cafe: Cafe) -> str:
```